build/build.tcl

```diff
@@ -664,6 +664,12 @@ expect ":KILL"
 respond "*" ":midas device;atsign rmtdev_gz;rmtdev\r"
 expect ":KILL"
 
+# spell
+respond "*" ":midas sys1;ts spell_syseng;spell\r"
+expect ":KILL"
+respond "*" ":link sys1;ts espell,sys;ts spell\r"
+respond "*" ":link .info.;spell info,info;spell >\r"
```

Comment:
I suppose you added the .INFO. link to mimic PI.  I see the link is used in INFO; DIRLOS 163.  But so far, we don't include the DIRLOS file, and I think it's just an older version.  So maybe that link isn't needed.